index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo">Dhuruv</div>
+        <div>
+            <img class="nav-icon" src="/Icons/Enabled.png" alt="nav button">
+        </div>
+    </header>
+    <main>
+        <section>
+            <h1>
+                Hey There.<br>I'm Dhuruv
+            </h1>
+            <h6>I'am a Software Developer</h6>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <ul>
+                <li>
+                    <a href="#">
+                        <img src="/Icons/Social_icon/GitHub.png" alt="Github icon">
```

Comment:
Also here in all the icons, you forgot the dot please, try to add it

Reply:
ok